app/src/main/java/com/xmartlabs/template/model/BuildInfo.kt

```diff
@@ -3,13 +3,8 @@ package com.xmartlabs.template.model
 import com.xmartlabs.template.BuildConfig
 import com.xmartlabs.bigbang.core.model.BuildInfo as CoreBuildInfo
 
-class BuildInfo : CoreBuildInfo {
-  override val isDebug: Boolean
-    get() = BuildConfig.DEBUG
-
-  override val isStaging: Boolean
-    get() = BuildConfig.FLAVOR == BuildType.STAGING.toString()
-
-  override val isProduction: Boolean
-    get() = BuildConfig.FLAVOR == BuildType.PRODUCTION.toString()
+object BuildInfo : CoreBuildInfo {
+  override val isDebug = BuildConfig.DEBUG
```

Comment:
That's because like in swift you don't have to specify the type of the variable that you are defining isn't it?